src/nls/root/menu.json

```diff
@@ -210,7 +210,7 @@
         "BRING_ALL_TO_FRONT": "Bring All to Front",
         "NEXT_DOCUMENT": "Next Document",
         "TOGGLE_TOOLBAR": "Pin Toolbar",
-        "TOGGLE_SINGLE_COLUMN_MODE": "Single Column Mode",
+        "TOGGLE_SINGLE_COLUMN_MODE": "Single-Column Mode",
```

Comment:
I didn't want to have to do this, but you forced my hand with that whole "no appearance or effects panel without a layer selection" thing.